catalog/solr/catalog-solr-core/src/main/java/ddf/catalog/source/solr/SolrMetacardClientImpl.java

```diff
@@ -150,6 +151,11 @@
   private final int commitNrtCommitWithinMs =
       Math.max(NumberUtils.toInt(accessProperty(SOLR_COMMIT_NRT_COMMITWITHINMS, "1000")), 0);
 
+  private static final String SOLR_QUERY_TIMEALLOWEDMS = "solr.query.timeAllowed";
+
+  private final int queryTimeAllowedMs =
+      Math.max(NumberUtils.toInt(accessProperty(SOLR_QUERY_TIMEALLOWEDMS, "0")), 0);
```

Comment:
❓ If you don't get a new Solr metacard client do you have to refresh the bundle to validate the system property change?

Reply:
Just now saw that in the testing instructions. No big deal.